src/cmd/ksh93/features/options

```diff
@@ -62,8 +62,17 @@ tst	note{ can we probe file system case insensitivity }end output{
 	static char *o = "SHOPT_GLOBCASEDET";
 	int main(int argc,char *argv[])
 	{
+		char *dir;
 		int r;
-		r = pathicase("/");
+		/* Prioritize /boot/EFI and /boot since those could
```

Comment:
Are these additions actually necessary? Note the feature test only returns false if `pathicase` returns -1 **and** sets `errno` to `ENOSYS`. According to the man page I found on the web, `ioctl`(2) on Linux does not ever set `errno` to `ENOSYS`.

Reply:
I just tested it and it turns out that change was unnecessary. Now reverted.